cypress/integration/Fields/PDQCitations/index.js

```diff
@@ -0,0 +1,52 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+//This step is just a workaround until it gets fixed
```

Comment:
reference this in the same ticket you will be creating to track these - this step should be removed after issue is fixed

Reply:
Done. Entered #118